projects_test.go

```diff
@@ -39,6 +39,16 @@ func TestAccProject(t *testing.T) {
 	if gotProject.PaymentMethod.URL == "" {
 		t.Fatalf("Empty payment_method: %v", gotProject)
 	}
+
+	events, _, err := c.Projects.ListEvents(gotProject.ID, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(events) == 0 {
```

Comment:
When I run the TestAccProjectBasic test, there is a HTTP request to 
```
GET /projects/3e0ab4ae-ed83-4f1d-9936-235a205029be/events?
```
and the returned events list is empty:
```
{"events":[],"meta":{"first":{"href":"/projects/3e0ab4ae-ed83-4f1d-9936-235a205029be/events?page=1"},"previous":null,"self":{"href":"/projects/3e0ab4ae-ed83-4f1d-9936-235a205029be/events?page=1"},"next":null,"last":{"href":"/projects/3e0ab4ae-ed83-4f1d-9936-235a205029be/events?page=0"},"current_page":1,"last_page":0,"total":0}}
```

I am not sure what is the proper behavior - should there be any events for new project? If not, the assert should inverted.

Reply:
I think it is safe to remove this check, because there won't be any events before a device is created.

Reply:
ok